### DISCUSSION TITLE
Red Dot on two of my installs

### DISCUSSION THREAD

**Discussion by rufusrizzo** [n1kdo/n1mm_view#63] [Q&A]
![sections_worked_map](https://github.com/n1kdo/n1mm_view/assets/6411830/63bcae3c-2122-41c6-9eb8-1d1a737661e9)
Two of my installs have a red dot in North GA.  

Is that something I can configure my home location?

**Comment by n1kdo** [comment]
Hi. Yes. Edit the config.py and set your lat,lon.

73,

Jeff

**Reply by rufusrizzo** [reply]
Thanks!  I found all sorts of knobs I can turn!  I was wondering why it was trying to copy files to something named "sparc"
